openfl-workspace/tf_cnn_histology/src/taskrunner.py

```diff
@@ -96,13 +96,30 @@ def create_model(self,
             metrics=[tf.keras.metrics.SparseCategoricalAccuracy()],
         )
 
-        self.tvars = model.layers
-        print(f'layer names: {[var.name for var in self.tvars]}')
+        return model
 
-        self.opt_vars = self.optimizer.variables()
-        print(f'optimizer vars: {self.opt_vars}')
+    def train_(self, batch_generator, metrics: list = None, **kwargs):
```

Comment:
In this particular case, could we consider not overriding the base `_train()` function, as it practically does the same (incl. additional validity checks which are missing here)?